@xen-orchestra/lite/src/types/new-vm.ts

```diff
@@ -20,7 +20,7 @@ export interface VmState {
   toggle: boolean
   installMode: string
   tags: string[]
-  affinity_host: string
+  affinity_host: XenApiHost['$ref'] | undefined
```

Comment:
If a template doesn't have an affinity host, it will return an empty opaque ref (and not undefined)
```suggestion
  affinity_host: XenApiHost['$ref'] | OPAQUE_REF.EMPTY
```